cmd/go-ycsb/client.go

```diff
@@ -55,6 +55,16 @@ func newWorker(p *properties.Properties, threadID int, threadCount int) *worker
 		}
 	}
 
+	if totalOpCount < int64(threadCount) {
+		msg := fmt.Sprintf("totalOpCount(%s/%s/%s): %d should be bigger than threadCount: %d",
+			prop.OperationCount,
+			prop.InsertCount,
+			prop.RecordCount,
+			totalOpCount,
+			threadCount)
+		panic(msg)
```

Comment:
I think panicking is too aggressive, how about printing the help message then exiting with non-zero status?